include/glow/Support/Half.h

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_SUPPORT_HALF_H
+#define GLOW_SUPPORT_HALF_H
+
+#include <iostream>
```

Comment:
no iostream please.

Reply:
Yeah that's because of EXPECT_EQ.
I'll dig to see how I can avoid it.

Reply:
Found a way around. The default printer prints the raw bytes of the class and use a cast to long long to do that.
Added the cast and now we don't need iostream anymore. (Though that prints the raw bytes)